test/spec/css/transformHEXRGBa.spec.ts

```diff
@@ -103,4 +103,30 @@ describe('transform HEXRGBa', () => {
       ).to.eql({ width: '2px', style: 'solid', color: 'rgba(0, 0, 0, 0.5)' });
     });
   });
+
+  it('transforms shorthand hex formats correctly', () => {
+    // 3-digit hex (#RGB)
+    expect(transformHEXRGBaForCSS({ value: '#F00' })).to.equal('#F00');
+
+    // 4-digit hex (#RGBA)
+    expect(transformHEXRGBaForCSS({ value: '#F00F' })).to.equal('rgba(255, 0, 0, 1)');
+    expect(transformHEXRGBaForCSS({ value: '#F000' })).to.equal('rgba(255, 0, 0, 0)');
```

Comment:
I would expect these to have the same output as input, since it's valid hex format.